app/javascript/packs/followButtons.js

```diff
@@ -51,23 +51,30 @@ function addButtonFollowText(button, style) {
  */
 function addAriaLabelToButton({ button, followType, followName, style = '' }) {
   let label = '';
+  let pressed = '';
   switch (style) {
     case 'follow':
       label = `Follow ${followType.toLowerCase()}: ${followName}`;
+      pressed = 'false';
       break;
     case 'follow-back':
       label = `Follow ${followType.toLowerCase()} back: ${followName}`;
+      pressed = 'true';
       break;
     case 'following':
       label = `Unfollow ${followType.toLowerCase()}: ${followName}`;
```

Comment:
Now that we're introducing the toggle button functionality, we don't need to update the aria-label to "Unfollow" anymore (🎉 ). This is because the "pressed" state tells the user if the button is currently activated (following) or not.  For this reason, updating both the label and the pressed state can be confusing for users.

See what you think is best, but we could either skip setting the aria-label for this case, or explicitly set it to `Follow ${followType.toLowerCase()}: ${followName}` 🙂

Reply:
@aitchiss Thanks, I completely forgot about the cypress tests, will update it soon.

Reply:
@aitchiss I've been thinking about the same thing, because in the previous PR #14256  where we set the aria-label for user comment reactions, once toggle functionality was added we made the **aria-label** static and having it dynamic here  was confusing me. Thanks for clearing this up. 

What about the Follow back functionality then? Do we need to implement that anymore as aria-pressed will be set to true so changing aria-label there also doesn't make sense.